src/traces/box/calc.js

```diff
@@ -122,7 +123,17 @@ module.exports = function calc(gd, trace) {
             num: fullLayout[numKey],
             dPos: dPos,
             posLetter: posLetter,
-            valLetter: valLetter
+            valLetter: valLetter,
+            labels: {
+                med: _(gd, 'median'),
+                min: _(gd, 'min'),
+                q1: _(gd, 'q1'),
+                q3: _(gd, 'q3'),
+                max: _(gd, 'max'),
+                mean: trace.boxmean === 'sd' ? _(gd, 'mean ± σ') : _(gd, 'mean'),
```

Comment:
also not sure about making `mean` and `mean ± σ` separate translations... most languages will probably leave ` ± σ` intact, but some might want to change it (Greek?)